### ISSUE TITLE
Cannot access menu from file manager

### ISSUE THREAD

**Issue by protist** [koreader/koreader#2830]
* KOReader version: koreader-nightly-20170430
* Device: Kobo H2O

#### Issue
I just upgraded to the latest version of koreader. I understand that while reading, the menu is now accessed by swiping down, and the tap zone is no longer functional. This works for me, however, I can't find a way to access the menu from the file manager. I've tried tapping the top, and swiping, and neither brings up the menu.

As an aside, the [wiki](https://github.com/koreader/koreader/wiki/Getting-Started) should be modified to refer to the new swipe trigger. Also the shipped `defaults.lua` still suggests `DTAP_ZONE_MENU`, which appears to be ignored.

#### Steps to reproduce
1. Open the file manager.
2. Attempt to access the menu.

**Comment by Frenzie** [comment]
That's very odd. None of what you say (except for the wiki, updated) should be true, nor is it on my own H2O. In fact the file manager menu *used* to ignore the `DTAP_ZONE_MENU` menu setting until this change, see https://github.com/koreader/koreader/commit/89099e2c940847442925d0f771f2ce95f133123e#diff-1163d9f66e4944329c096f149b66085fR58

**Comment by a-kohout** [comment]
Did you change `DTAP_ZONE_MENU`?
Before the update only the reader used the menu tap zone, the file manager would always trigger the menu from the top.
Since my menu tap zone used to be in the vertical middle third of the screen, I had to swipe there after the update.
I have since changed that because swiping in the middle of the screen is really confusing.

**Comment by a-kohout** [comment]
@Frenzie Since the quickstart guide is generated on the device, should it mention where to swipe for configurations where `DTAP_ZONE_MENU` deviates from the default?

**Comment by protist** [comment]
Thank you to you both. I tried again, and it's now working! No idea what the problem was before.

@Frenzie I originally had `DTAP_ZONE_MENU` set in the middle quarter of the screen, but I had turned it off when I tested the swipe zone. Nevertheless, it didn't work the first time, although it's fine now.

@a-kohout Thanks for that advice. I agree, that *is* really confusing. I had reverted back to `DTAP_ZONE_MENU` in the middle quarter of the screen, and I was confused as to why I had to swipe such a ridiculous length to get the menu to work, and why it often failed. I agree that this definitely needs a little more direction for users.

I'm happy to close this issue from my perspective, but I'll leave it open for now, in case you want to mention the function of `DTAP_ZONE_MENU` regarding swiping. Thanks again for the quick and useful comments.

**Comment by a-kohout** [comment]
@protist You don't have to swipe all the way. A short swipe south (e.g. 1.5cm/0.6in) anywhere inside the menu zone will do.

**Comment by protist** [comment]
@a-kohout That issue was when I had `DTAP_ZONE_MENU` defined in the middle quarter of the screen. I didn't realise that this setting was relevant for the swipe. I thought the swipe still worked from the top of the screen. Hence, the menu didn't appear when I'd swipe ~5 cm starting at the top, but it did when I swiped down more than halfway through the page. In retrospect, this was because I was unknowingly swiping through the `DTAP_ZONE_MENU`. When I realised how the setting/swipe worked, I could indeed just swipe 1.5 cm within this zone.
